app/services/dttp/placement_assignment.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Dttp
+  class PlacementAssignment
+    include ServicePattern
+
+    class HttpError < StandardError; end
+
+    attr_reader :trainee
+
+    PLACEMENT_ASSIGNMENT_FIELDS = %w[
+      dfe_programmestartdate
+      dfe_programmeenddate
+      dfe_placementassignmentid
+      _dfe_providerid_value
+    ].freeze
+
+    def initialize(trainee:)
+      @trainee = trainee
+    end
+
+    def call
+      response = Client.get("/dfe_placementassignments(#{trainee.placement_assignment_dttp_id})?$select=#{PLACEMENT_ASSIGNMENT_FIELDS.join(',')}")
```

Comment:
I think we should just get all fields here. Otherwise we're going to have to maintain a version of this list in a few places.

Reply:
fixed, cheers